nextflow_schema.json

```diff
@@ -255,6 +255,18 @@
                     "description": "Run bcftools call in either consensus or multiallelic mode",
                     "enum": ["consensus", "multiallelic"],
                     "help_text": "Bcftools call can eitherbe run in multiallelic mode or in consensus mode. In consensus mode a p-value threshold of 0.01 is applied."
+                },
+                "run_drop_ae": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "Should DROP Aberrant Expression module be run?",
+                    "fa_icon": "fas fa-toggle-off"
+                },
+                "reference_count_file": {
+                    "type": "string",
+                    "default": null,
+                    "description": "If you are running  DROP Aberrant Expression, provide the path to a tsv with counts from at least 50 samples to use as controls",
```

Comment:
you can add: "format": "file-path", via the schema builder

Reply:
THat's true! Done!